src/sentry/utils/avatar.py

```diff
@@ -125,3 +126,12 @@ def get_email_avatar(
                     gravatar_url = get_gravatar_url(identifier, size=size)
                     return f'<img class="avatar" src="{gravatar_url}">'
     return get_letter_avatar(display_name, identifier, size, use_svg=False)
+
+
+def is_black_alpha_only(data: Any) -> bool:
```

Comment:
Is there not a better type we can use here?

Reply:
:( lol idk how to type an image, let me try to figure it out

Reply:
Look at what data type BytesIO takes